admin/app/views/fragments/abTestReport.scala.html

```diff
@@ -1,20 +1,12 @@
 @()
 <div class="abtests-report">
     <div class="col-md-12 abtest__details">
-        <table class="abtest__table table table-r table-bordered table-hover">
-            <thead>
-                <th>Name</th>
-                <th>State</th>
-                <th>Variants</th>
-                <th class="abtest-item__expiry-title">Expires</th>
-                <th class="abtest-item__audience-title">Audience</th>
-                <th class="abtest-teim__audience-offset-title">Offset</th>
-                <th>Tableau</th>
-                <th>Ophan</th>
-            </thead>
-            <tbody class="abtests-report__data">
-                <!-- abTestReportItem instance(s) will be written here -->
-            </tbody>
+        <table class="abtest__table table table-r table-bordered">
+            <tr>
+                <td class="abtests-report__data">
+                    <!-- report items will be added here -->
```

Comment:
maybe use a Play comment `@* report items will be added here *@` then it won't end up in HTML